traits/trait_type.pyi

```diff
@@ -28,9 +28,6 @@ class _TraitMixin(Generic[_Accepts, _Stores]):
     def __set__(self, object: Any, value: _Accepts) -> None: ...
 
 
-_Trait = _TraitMixin()
```

Comment:
I can't tell what this was for, and mypy was complaining about `_Trait` needing a type declaration, so I've deleted it.